### DISCUSSION TITLE
new UI design for Gists: Unsubcribe button is now where Edit gists button used to be (causing unwanted clicks)

### DISCUSSION THREAD

**Discussion by unitycoder** [community/community#119698] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

years of muscle memory is causing unwanted clicks to unsubscribe button (for your own gists).
Not to mention that 99% of the time you never need to click "unsubscribe button" there, so would
be nice to have Edit button in the original position.

new design: (notice that unsub. button is first.. <sad face> )
![image](https://github.com/community/community/assets/5438317/91c2bed9-80b0-4c49-8334-4fd34670d27f)

**Comment by wilmoore** [comment]
The other issue with this and other changes to gists is that they just seem to appear out of thin air and there are no announcements/changelog entries about the change like there are with changes to other areas of GitHub.

**Reply by unitycoder** [reply]
yeah.. i reported this as a bug,
and even the support didn't know this update was getting rolled out!
went through 5 emails, they asked to try different browser, record video clips, screenshots, until they realized:

> "I can see the unsubscribe button on the left when I check, so I think you were correct that a change to the UI is being rolled out."
